packages/terra-slide-panel/src/terra-dev-site/doc/slide-panel/SlidePanel.1.doc.jsx

```diff
@@ -28,7 +27,7 @@ import SlidePanelLarge from '../../test/slide-panel/SlidePanelLarge.test';
 import SlidePanelFullscreen from '../../test/slide-panel/SlidePanelFullscreen.test';
 import SlidePanelFill from '../../test/slide-panel/SlidePanelFill.test';
 import SlidePanelNoFill from '../../test/slide-panel/SlidePanelNoFill.test';
-/* eslint-enabled import/no-webpack-loader-syntax, import/first, import/extensions, import/no-unresolved, import/no-duplicates */
+
```

Comment:
nit: I am just nit picking here, but you could remove this extra space just for consistency, like as in the other files.